packages/integration/report.api.md

```diff
@@ -398,13 +399,13 @@ export type GerritIntegrationConfig = {
 };
 
 // @public
-export function getAzureCommitsUrl(url: string): string;
+export function getAzureCommitsUrl(url: string, apiVersion: string): string;
```

Comment:
We can't make this a breaking change. The library is at a 1.x version and would have required a major release to do this.

The new field in `AzureIntegrationConfig` is in a grey area. It SHOULD only affect producers (not people who read the type) which hopefully are none. So we might let that slide.

However, these three functions should be refactored to have an _optional_ second argument `options?:  { apiVersion?: string }` to not break the contract.